_posts/en/newsletters/2020-09-30-newsletter.md

```diff
@@ -0,0 +1,189 @@
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What is the history of nLockTime in Bitcoin Core?]({{bse}}90229)
+  Reviewing older versions of Bitcoin Core's source code, Martin Harrigan asks
+  for clarification on whether height-based or time-based nLockTime features
+  were added first and any fork implications. Pieter Wuille and David A. Harding
+  confirm height-based logic came first and give some interesting historical insights.
+
+- [Is there any other P2P protocol in use besides a "Gossip protocol"?]({{bse}}99131)
+  Murch outlines the different communication protocols used in the Bitcoin
+  ecosystem including [Bitcoin's P2P protocol][bitcoin p2p messages], mining
+  protocols like FIBRE and stratum, LN-related protocols, as well as multiparty
+  coordination protocols like [payjoin][topic payjoin].
+
+- [Why do Anchor Outputs need to enforce an nSequence of 1?]({{bse}}98848)
+  Dalit Sairio asks about [anchor outputs][topic anchor outputs] and the
+  necessity of the OP_CHECKSEQUENCEVERIFY (CSV) addition to LN scripts. User
+  darosior describes the concerns around [CPFP carve-out][topic cpfp carve out]
+  and why the delay of 1 block is needed.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.1-beta.rc4][lnd 0.11.1-beta] is the release candidate for a
+  minor version.  Its release notes summarize the changes as "a number
+  of reliability improvements, some macaroon [authentication token]
+  upgrades, and a change to make our version of [anchor commitments][topic
+  anchor outputs] spec compliant."
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #18267][] and [#19993][Bitcoin Core #19993] add support
+  for [signet][topic signet].  If Bitcoin Core is started with `-signet`
+  (or `-chain=signet`), it'll connect to either the default signet or
+  the signet defined by the `-signetchallenge` and `-signetseednode`
+  parameters.
+
+- [Bitcoin Core #19572][] adds a new "sequence" ZMQ topic that notifies
```

Comment:
Suggest linking to https://github.com/bitcoin/bitcoin/blob/master/doc/zmq.md for users to learn more